zwave_js_server/util/command_class.py

```diff
@@ -0,0 +1,55 @@
+"""Command Class specific utility functions for values."""
+from typing import Union
+
+from ..const import (
+    CC_SPECIFIC_METER_TYPE,
+    CC_SPECIFIC_SCALE,
+    CC_SPECIFIC_SENSOR_TYPE,
+    METER_TYPE_TO_SCALE_ENUM_MAP,
+    CommandClass,
+    CoolingScale,
+    ElectricScale,
+    GasScale,
+    HeatingScale,
+    MeterType,
+    MultilevelSensorType,
+    WaterScale,
+)
+from ..exceptions import InvalidCommandClass
+from ..model.value import Value
+
+
+def get_meter_type(value: Value) -> Union[MeterType, None]:
+    """Get the MeterType for a given value."""
+    if value.command_class != CommandClass.METER:
+        raise InvalidCommandClass(value, CommandClass.METER)
+
+    try:
+        return MeterType(value.metadata.cc_specific[CC_SPECIFIC_METER_TYPE])
+    except ValueError:
+        return None
+
+
+def get_meter_scale_type(
+    value: Value,
+) -> Union[CoolingScale, ElectricScale, GasScale, HeatingScale, WaterScale, None]:
+    """Get the ScaleType for a given value."""
+    meter_type = get_meter_type(value)
+    if meter_type is None:
+        return None
+    scale_enum = METER_TYPE_TO_SCALE_ENUM_MAP[meter_type]
+    try:
+        return scale_enum(value.metadata.cc_specific[CC_SPECIFIC_SCALE])  # type: ignore
```

Comment:
What type error are we ignoring?

Reply:
`zwave_js_server/util/command_class.py:43: error: Returning Any from function declared to return "Union[HeatingScale, ElectricScale, GasScale, HeatingScale, WaterScale, None]"`

I tried type hinting scale_enum with `IntEnum` and `EnumMeta` and that didn't help

Reply:
With the type hint fix below I was able to remove this